shell/shared/admin.js

```diff
@@ -46,6 +46,29 @@ var adminRoute = RouteController.extend({
   },
 
   action: function () {
+    // Test the WILDCARD_HOST for sanity.
+    Deps.nonreactive(function() {
+      if (Session.get("alreadyTestedWildcardHost")) {
+        return;
+      }
+      let req = new XMLHttpRequest();
```

Comment:
Meteor's `HTTP` package has a much nicer interface. Consider using it rather than using XHR directly?